packages/gatsby/src/redux/actions/public.js

```diff
@@ -423,6 +423,7 @@ ${reservedFields.map(f => `  * "${f}"`).join(`\n`)}
     // Ensure the page has a context object
     context: page.context || {},
     updatedAt: Date.now(),
+    manifestId: ``,
```

Comment:
@KyleAMathews @TylerBarnes Should this stay as-is or should it be `page.manifestId || ''`?

Reply:
@LekoArts your suggestion looks right to me